web/docs/project/env-vars.md

```diff
@@ -104,19 +104,20 @@ Defining environment variables in this way can be cumbersome even for a single p
 
 ## Defining Env Vars in Production
 
-While in development, we had the option of using `.env` files which made it easy to define and manage env vars. However, in production, we need to provide env vars differently.
-
 ![Env vars usage in development and production](/img/env/prod_dev_fade_2.svg)
 
-### Client Env Vars
+While in development, we had the option of using `.env.client` and `.env.server` files which made it easy to define and manage env vars.
```

Comment:
Moved this below the image because image is big enough that I failed to see this text multiple times while reading the stuff below (to which I arrived by hyperlink).